docs/design/001-interchain-queries.md

```diff
@@ -0,0 +1,128 @@
+# Design Doc 001: Interchain Queries
+
+## Status
+
+Draft
+
+## Abstract
+
+In order for the Ojo network to effectively provide services to other chains, we need a method of transport at Layer 1. We must develop two mechanisms: a *Client* module for consumer chains to use; and a *Relayer* to move messages in between The Ojo chain and the *Client* chain. The Client chain will request data, which the relayer will then pick up, query Ojo for the requested data, and then respond back to the original chain.
+
+*Client* Module - accepts validator responses to requests made by the *Client*; requires a *Proof* for information to be accepted.
+
+*Relayer* - allows relayers to move information between the *Host* and *Client*, and is responsible for submitting proofs.
+
```

Comment:
Would it make sense to have the 1 sentence description of the *host* module still here even if we're creating a separate design doc to spec it out just to relieve any confusion since its referenced throughout this doc?  If I understand correctly we still need it even we're not implementing the encryption and economic factor yet.

Reply:
In this implementation (without encryption / rewards) we remove the host module altogether for now. I think it'd be better for me to remove references to it @rbajollari